ci-operator/step-registry/openshift/e2e/openstack/ipi/openshift-e2e-openstack-ipi-workflow.yaml

```diff
@@ -14,6 +14,7 @@ workflow:
       CONFIG_API_VERSION: "v1beta4"
       CONFIG_TYPE: "minimal"
       BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      IGNITIONVERSION: "2.2.0"
```

Comment:
Testing shows that openshift is able to handle manifests files with previous ignition versions. MCO merges the manifest files into a machineset with the latest ignition version
Thus setting it here to 2.2.0 will allow this workflow to be used for OpenShift versions 4.3, 4.4, 4.5, 4.6, and 4.7
I already tested this by hand and as I said MCO merges the manifest file into the "xxxx-merged master" machineconfig. for the masters.